QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -91,7 +91,8 @@ private static void setupRollers() {
         BooleanMixing.pumpWhen(QuasarHelios.manualControl, BooleanMixing.andBooleans(rollersMode, rightTriggerPress), Rollers.rightPneumaticOverride);
 
         BooleanMixing.pumpWhen(QuasarHelios.manualControl,
-                BooleanMixing.andBooleans(rollersMode.asInvertedInput(), FloatMixing.floatIsAtLeast(leftTrigger, cutoffAuto)), QuasarHelios.autoEjector);
+                BooleanMixing.andBooleans(rollersMode.asInvertedInput(), FloatMixing.floatIsAtLeast(leftTrigger, cutoffAuto),
+                        QuasarHelios.autoHumanLoader.asInvertedInput()), QuasarHelios.autoEjector);
```

Comment:
Maybe an informative comment about the meaning of these lines would be helpful. (Took me a minute to figure out.)